source/get-started.md

```diff
@@ -82,7 +82,7 @@ $ cd themes/doc && npm install --prod
 
 Install the required hexo plugins in your project:
 ```
-$ npm install hexo-renderer-ejs hexo-renderer-marked --save
+$ npm install hexo-renderer-ejs hexo-renderer-marked hexo-renderer-sass hexo-renderer-less --save
```

Comment:
@noraj Why do we need also to install less and sass, that is only required if you actually use one those two or am I missing something?

Reply:
Yeah but I saw that here: https://github.com/zalando-incubator/hexo-theme-doc-seed/blob/master/package.json

Reply:
Ah cool, it's a dependency of the seed which is meant to get you started quickly using all the features of hexo-theme-doc, I will keep just `hexo-renderer-ejs hexo-renderer-marked`. SASS and LESS are required just if you want to provide your extra styles using pre-processors. BTW thanks for your contribution.